File: ts/iq/w-g2~set~sub_group_suggestion.ts
/** <iq type="set" xmlns="w:g2">. Replies with one of: NewGroupSuggestionSuccess, ExistingGroupsSuggestionSuccess, ClientError, ServerError. */
export function buildGroupsCreateSubGroupSuggestion(to: string, to: string): BinaryNode {
  return { tag: 'iq', attrs: { to: to, xmlns: 'w:g2', type: 'set' }, content: [
      { tag: 'sub_group_suggestion', attrs: {  } },
    ] }
}

File: go/iq/w-g2~set~sub_group_suggestion.go
// BuildGroupsCreateSubGroupSuggestion builds <iq type="set" xmlns="w:g2">. Replies with one of: NewGroupSuggestionSuccess, ExistingGroupsSuggestionSuccess, ClientError, ServerError.
func BuildGroupsCreateSubGroupSuggestion(to types.JID, to types.JID) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"to": to, "xmlns": "w:g2", "type": "set"},
		Content: []waBinary.Node{
			{Tag: "sub_group_suggestion", Attrs: waBinary.Attrs{}},
		},
	}
}